security/integration/main.go

```diff
@@ -276,16 +337,18 @@ func examineSecret(secret *v1.Secret, expectedID string) error {
 		}
 	}
 
-	key := secret.Data[controller.PrivateKeyID]
-	cert := secret.Data[controller.CertChainID]
-	root := secret.Data[controller.RootCertID]
+	// should be valid
```

Comment:
Still confused by the comment - *what* should be valid?

Reply:
Removed